packages/consumption/test/modules/requests/itemProcessors/proposeAttribute/ProposeAttributeRequestItemProcessor.test.ts

```diff
@@ -375,33 +388,59 @@ describe("ProposeAttributeRequestItemProcessor", function () {
                 statusLog: []
             });
 
-            const acceptParams: AcceptProposeAttributeRequestItemParametersJSON = {
+            const acceptParams: AcceptProposeAttributeRequestItemParametersWithExistingAttributeJSON = {
                 accept: true,
-                attributeId: idOfAttributeOfOtherIdentity.toString()
+                attributeId: attribute.id.toString()
             };
 
             const result = await processor.canAccept(requestItem, acceptParams, request);
 
             expect(result).errorValidationResult({
-                code: "error.consumption.requests.invalidRequestItem",
-                message: /The given Attribute belongs to someone else. You can only share own Attributes./
+                code: "error.consumption.requests.attributeQueryMismatch",
+                message: "The provided IdentityAttribute is a shared copy of a RepositoryAttribute. You can only share RepositoryAttributes."
             });
         });
 
-        test("returns an error when the given Attribute's owner is not the current identity", async function () {
-            const someOtherIdentity = CoreAddress.from("id1");
+        test("returns an error when a Successor of the existing IdentityAttribute is already shared", async function () {
```

Comment:
This test moved as "returns an error when the given Attribute's owner is not the current identity" to the file validateAttributeMatchesWithQuery.test.ts in the describe block "describe("ProposeAttributeRequestItem with IdentityAttributeQuery")", because this error is now thrown by the auxiliary function validateAttributeMatchesWithQuery.